scripts/create-agent.ts

```diff
@@ -251,7 +251,21 @@ class VargasJRAgentCreator {
     
     try {
       const dbName = this.config.name.replace('-', '_');
-      const envContent = `POSTGRES_URL=postgresql://postgres:password@localhost:5432/vargasjr_${dbName}
+      
+      let postgresUrl: string;
+      if (this.config.prNumber) {
+        try {
+          postgresUrl = await getNeonPreviewDatabaseUrl(this.config.prNumber);
+        } catch (error) {
+          console.warn(`⚠️  Failed to get Neon database URL for preview agent: ${error}`);
+          console.log("Falling back to local PostgreSQL setup");
+          postgresUrl = `postgresql://postgres:password@localhost:5432/vargasjr_${dbName}`;
```

Comment:
Don't even bother falling back to local. Fail the script if we can't find the Postgres url